juju/controller.py

```diff
@@ -63,6 +63,20 @@ class Controller(object):
 
         return model
 
+    async def destroy_models(self, *args):
+
+        """Destroy a model to this controller.
+
+        :param str : model-<UUID>
+
+        """
+        model_facade = client.ModelManagerFacade()
+        model_facade.connect(self.connection)
+
+        for arg in args:
+            log.debug('Destroying Model %s', arg)
+            await model_facade.DestroyModels([client.Entity(arg)])
```

Comment:
I would suggest making a list of client.Entity objects from args so that only one call to DestroyModels is needed.